packages/interaction-kit/src/commands/options/option.ts

```diff
@@ -7,12 +7,9 @@ import {
 	APIApplicationCommandInteractionDataStringOption,
 	APIApplicationCommandOption,
 	ApplicationCommandOptionType,
-} from "discord-api-types/payloads/v9";
-import { APIApplicationCommandOptionBase } from "discord-api-types/payloads/v9/_interactions/_applicationCommands/_chatInput/base";
+} from "discord-api-types/v10";
+import { APIApplicationCommandOptionBase } from "discord-api-types/payloads/v10/_interactions/_applicationCommands/_chatInput/base";
```

Comment:
Did we not export these at the top level yet?

Reply:
Not yet no

Reply:
I thought you PR'd it 🤔